src/cmd_line.rs

```diff
@@ -50,10 +50,13 @@ pub struct CmdLineSettings {
     #[arg(long = "no-multigrid", env = "NEOVIDE_NO_MULTIGRID", value_parser = FalseyValueParser::new())]
     pub no_multi_grid: bool,
 
-    /// Instead of spawning a child process and leaking it, be "blocking" and let the shell persist
-    /// as parent process
-    #[arg(long = "no-fork")]
-    pub no_fork: bool,
+    /// Spawn a child process and leak it
```

Comment:
Since `_no_fork` is not public, its documentation comment should be included here as well, so it's also displayed in the `--help`.

Reply:
I'll do that right now and then merge, actually.

Reply:
Nvm I misunderstood the arguments. I clarified the help anyway.